index.d.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  Readable,
+  ReadableOptions,
+  Writable,
+  WritableEvents,
+  WritableOptions
+} from 'bare-stream'
+
+interface ReadStream extends Readable {
+  readonly isTTY: true
+
+  setMode(mode: boolean): this
+
+  setRawMode(mode: boolean): this
```

Comment:
```suggestion
  setRawMode(enabled: boolean): this
```

@yassernasc Can you rename the argument in the implementation as well? It makes things a bit clearer.